File: examples/python/stories.py
from e3d_client import E3DClient


def main() -> None:
    client = E3DClient()
    stories = client.get("/stories", query={"limit": 5})
    addresses = client.get("/stories/addresses")

    print(f"Stories: {len(stories) if isinstance(stories, list) else 'unknown'}")
    print(f"Addresses sample: {addresses[:3] if isinstance(addresses, list) else addresses}")


if __name__ == "__main__":
    main()

File: examples/stories.mjs
import { E3D } from '../dist/index.js';

const e3d = new E3D({ baseUrl: process.env.E3D_BASE_URL || 'https://e3d.ai/api' });
const stories = await e3d.stories.list({ limit: 5 });
const addresses = await e3d.stories.getAddresses();

console.log('Stories:', stories.length);
console.log('Addresses sample:', addresses.slice(0, 3));
